test/spec/core/CanvasSpec.js

```diff
@@ -867,6 +867,25 @@ describe('Canvas', function() {
       }));
 
 
+      it('should return copy of viewbox', inject(function(canvas) {
+
+        // given
+        canvas.addShape({ id: 's0', x: 0, y: 0, width: 300, height: 300 });
+
+        var shape = canvas.addShape({ id: 's1', x: 10000, y: 0, width: 300, height: 300 });
+
+        var viewbox = canvas.viewbox(),
+            viewboxStringified = JSON.stringify(viewbox);
+
+        // when
+        canvas.scrollToElement(shape);
+
+        // then
+        expect(JSON.stringify(viewbox)).to.eql(viewboxStringified);
```

Comment:
I don't understand the test either :see_no_evil:. If we want the to verify "no identity", why don't we test for "not equal"?

Reply:
I'm testing whether the viewbox I got from calling `viewbox()` has been changed. If you remove the `JSON.parse(JSON.stringify())` part you'll see the test failing.

Reply:
A much simpler test would be:

```
// mind the "strict object equality"
expect(canvas.viewbox()).not.to.equal(canvas.viewbox())
```

But 

```
// mind the loose object comparison
expect(canvas.viewbox()).to.eql(canvas.viewbox());
```

?